Resources/Textures/_CP14/Structures/Furniture/wood_cupboard.rsi/meta.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

Comment:
Maybe we should use a new project license for all new sprites, if we don't want other people to use them without our permission. But that's up to you.

Reply:
Oh, yeah, that totally slipped my mind. I'll change it.


> Maybe we should use a new project license for all new sprites, if we don't want other people to use them without our permission. But that's up to you.

Oh, yeah, that totally slipped my mind. I'll change it.